api/v1alpha1/openapi.yaml

```diff
@@ -3432,11 +3432,82 @@ components:
         - metadata
         - items
       description: FleetList is a list of Fleets.
+
+    DisruptionAllowance:
+      type: object
+      properties:
+        groupBy:
+          type: array
+          items:
+            type: string
+          description: List of labels to perform grouping for the disruption allowance.
```

Comment:
Change to "selector" and use our LabelSelector definition

Reply:
This is group by keys.  It does not contain label values.

Reply:
I guess it depends on whether we want to allow selecting by group (key) or also specific location. With a selector that could be `store=Madrid` or just `store` (existence op). What would an example `groupBy` be? Would the list items be OR'ed or AND'ed?

Reply:
This is not a selector.  It is used for disruption allowance.
Its purpose is that we want to define (for example) that we don't want (for any site) that the number of unavailable devices will be greater than the unavailable devices as a result of a rollout.
If we want a specific site to be selected then it needs to be defined in the batch sequence definition.

Reply:
Think of it like the `group by` clause in aggregate SQL select statement.  Only field names are defined there.